salt/state.py

```diff
@@ -1642,7 +1644,7 @@ def requisite_in(self, high):
         errors.extend(req_in_errors)
         return req_in_high, errors
 
-    def call(self, low, chunks=None, running=None):
+    def call(self, low, chunks=None, running=None, retries=1):
```

Comment:
No, perhaps that parameter should be renamed to attempt_counter.  It will only be retried if the "retry" parameter is passed to the state ("retry" is in the low data).  The "retries" parameter to the call function tracks the number of retry attempts that have happened.   This is b/c "call" is recursively executed either until call's "retries" is greater than the number of attempts passed to the state's retry parameter or until the desired result is returned from the state.

``